app-tasks/rf/src/rf/uploads/landsat8/settings.py

```diff
@@ -0,0 +1,7 @@
+"""Settings shared by functions for indexing Landsat 8 data"""
+
+organization = 'dfac6307-b5ef-43f7-beda-b9f208bb7726'
+
+usgs_landsat_url = (
```

Comment:
Yeah definitely, especially since the failure would be _really_ far from where the scene was created.